### DISCUSSION TITLE
What a great quickapp !!

### DISCUSSION THREAD

**Discussion by LodeBo** [alexander-vitishchenko/hc3-to-mqtt#52] [General]
Thank you so much for this great quickapp !!!
I will definitely make a donation.
From the bottom of my heart, i hope you and your familiy can return to home soon and live a peacefull life.

Best regards,
Lode.

**Comment by alexander-vitishchenko** [comment]
Thank you Lode!

**Comment by Eroi69** [comment]
True, after update from 1.0.219 to 1.0.230 i finally got my electric meter kWh and all the other things Qubino smart meter gives me <3 Amazing app keep up the good work!
